README.md

```diff
@@ -25,47 +7,48 @@ This generator methodology enables the creation of re-usable components and libr
 
-Chisel is powered by [FIRRTL (Flexible Intermediate Representation for RTL)](https://github.com/chipsalliance/firrtl), a hardware compiler framework that performs optimizations of Chisel-generated circuits and supports custom user-defined circuit transformations.
-
-* [What does Chisel code look like?](#what-does-chisel-code-look-like)
-  * [LED blink](#led-blink)
-  * [FIR Filter](#fir-filter)
-* [Getting Started](#getting-started)
-  * [Bootcamp Interactive Tutorial](#bootcamp-interactive-tutorial)
-  * [A Textbook on Chisel](#a-textbook-on-chisel)
-  * [Build Your Own Chisel Projects](#build-your-own-chisel-projects)
-  * [Guide For New Contributors](#guide-for-new-contributors)
-  * [Design Verification](#design-verification)
-* [Documentation](#documentation)
-  * [Useful Resources](#useful-resources)
-  * [Chisel Dev Meeting](#chisel-dev-meeting)
-  * [Data Types Overview](#data-types-overview)
-* [Contributor Documentation](#contributor-documentation)
-  * [Useful Resources for Contributors](#useful-resources-for-contributors)
-  * [Compiling and Testing Chisel](#compiling-and-testing-chisel)
-  * [Running Projects Against Local Chisel](#running-projects-against-local-chisel)
-  * [Building Chisel with FIRRTL in the same SBT Project](#building-chisel-with-firrtl-in-the-same-sbt-project)
-  * [Chisel3 Architecture Overview](#chisel3-architecture-overview)
-  * [Chisel Sub-Projects](#chisel-sub-projects)
-  * [Which version should I use?](#which-version-should-i-use)
-  * [Chisel Roadmap](#roadmap)
+Chisel is powered by [FIRRTL (Flexible Intermediate Representation for RTL)](https://github.com/chipsalliance/firrtl-spec),
+a hardware compiler framework implemented by [LLVM CIRCT](https://github.com/llvm/circt).
+
+- [What does Chisel code look like?](#what-does-chisel-code-look-like)
+  - [LED blink](#led-blink)
+  - [FIR Filter](#fir-filter)
+- [Getting Started](#getting-started)
+  - [Bootcamp Interactive Tutorial](#bootcamp-interactive-tutorial)
+  - [A Textbook on Chisel](#a-textbook-on-chisel)
+  - [Build Your Own Chisel Projects](#build-your-own-chisel-projects)
+  - [Guide For New Contributors](#guide-for-new-contributors)
+  - [Design Verification](#design-verification)
+- [Documentation](#documentation)
+  - [Useful Resources](#useful-resources)
+  - [Chisel Dev Meeting](#chisel-dev-meeting)
+  - [Data Types Overview](#data-types-overview)
+- [Contributor Documentation](#contributor-documentation)
+  - [Useful Resources for Contributors](#useful-resources-for-contributors)
+  - [Compiling and Testing Chisel](#compiling-and-testing-chisel)
+  - [Running Projects Against Local Chisel](#running-projects-against-local-chisel)
+  - [Chisel3 Architecture Overview](#chisel3-architecture-overview)
+  - [Chisel Sub-Projects](#chisel-sub-projects)
+  - [Which version should I use?](#which-version-should-i-use)
+  - [Roadmap](#roadmap)
```

Comment:
VSCode auto regenerated this which is both annoying for the churn but nice for the guaranteed accuracy.